client/web/src/enterprise/codeintel/shared/components/CodeIntelIndexer.tsx

```diff
@@ -0,0 +1,8 @@
+import React, { FunctionComponent } from 'react'
+
+import { Link } from '@sourcegraph/wildcard'
+
+import { CodeIntelIndexerFields } from '../../../../graphql-operations'
+
+export const CodeIntelIndexer: FunctionComponent<{ indexer: CodeIntelIndexerFields }> = ({ indexer }) =>
+    indexer.url === '' ? <>{indexer.name}</> : <Link to={indexer.url}>{indexer.name}</Link>
```

Comment:
```suggestion
export const CodeIntelIndexer: FunctionComponent<{ indexer: CodeIntelIndexerFields }> = ({ indexer }) => <LinkOrSpan to={indexer.url}>{indexer.name}</LinkOrSpan>
```

is a bit nicer to read